transport-native-io_uring/src/main/java/io/netty/channel/uring/IOUringEventLoop.java

```diff
@@ -65,14 +63,10 @@
         // Ensure that we load all native bits as otherwise it may fail when try to use native methods in IovArray
         IOUring.ensureAvailability();
 
-        // TODO: Let's hard code this to 8 IovArrays to keep the memory overhead kind of small. We may want to consider
-        //       allow to change this in the future.
-        iovArrays = new IovArrays(8);
         ringBuffer = Native.createRingBuffer(ringSize, iosqeAsyncThreshold, new Runnable() {
             @Override
             public void run() {
-                // Once we submitted its safe to clear the IovArrays and so be able to re-use these.
-                iovArrays.clear();
+                // NOOP
```

Comment:
do we still need the submissionQueueCallback in the future?

Reply:
maybe... not sure yet. We can rip it out if we find no usage. It doesn't hurt atm

Reply:
let's remove it I guess or @njhill what do you think?

Reply:
ok let me do it...